.gitattributes

```diff
@@ -0,0 +1,2 @@
+go.mod    linguist-generated=true
```

Comment:
do we need this file?
the formatting looks inconsistent

Reply:
Removed go.mod from gitattributes as it is the dependencies file.
Formatted the file as it was not formatted indeed.